docs/cloud-native-security/get-started-with-kspm.asciidoc

```diff
@@ -236,3 +236,136 @@ The *Add agent* wizard helps you deploy the KSPM integration on the Kubernetes c
 2. Apply the manifest using the `kubectl apply -f` command. For example: `kubectl apply -f elastic-agent-managed-kubernetes.yaml`
 
 After a few minutes, a message confirming the {agent} enrollment appears, followed by a message confirming that data is incoming. You can then click *View assets* to see where the newly-collected configuration information appears throughout {kib}, including the <<findings-page,Findings page>> and the <<cloud-posture-dashboard, Cloud Posture dashboard>>.
+
+[discrete]
+[[kspm-eck]]
+=== Set up KSPM on ECK deployments
+When running https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-quickstart.html[ECK] deployment, KSPM benchmark will is not support out-of-the-box.
```

Comment:
```suggestion
To run KSPM on an https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-quickstart.html[ECK] deployment,
```

Reply:
Agree with @benironside's edit above, but it's not showing up as applied in the diff or in the rendered HTML preview. Not sure if this is related to other GitHub problems we've been having lately, but commenting here and unresolving the conversation for visibility.